src/highspy/_core/__init__.pyi

```diff
@@ -788,6 +788,7 @@ class _Highs:
         indices: numpy.ndarray[typing.Any, numpy.dtype[numpy.int32]],
         values: numpy.ndarray[typing.Any, numpy.dtype[numpy.float64]],
     ) -> HighsStatus: ...
+    def addLinearObjective(self, linear_objective: HighsLinearObjective) -> HighsStatus: ...
```

Comment:
@mathgeekcoder As discussed, I have removed addLinearObjective and clearLinearObjectives from highs.py. However, I think it's worth having them here for the IDEs. In VSCode for example, without this, the code still works but the function is white and not clickable. I would assume it's also important for mypy